### DISCUSSION TITLE
Question about choosing Javascript and not alternatives.

### DISCUSSION THREAD

**Discussion by karolzlot** [Koenkk/zigbee2mqtt#7641] [General]
@Koenkk, thank you for your work on zigbee2mqtt.

I am curious about your decision-making process.

What made you to use Javascript + NodeJS and not Python / Go / Rust / Dart or other technologies when you started this project?

**Comment by Koenkk** [comment]
I've started this project from https://github.com/AndrewLinden/xiaomi-zb2mqtt / and https://github.com/zigbeer/zigbee-shepherd which were written in Javascript/NodeJS
